README.md

```diff
@@ -1,141 +1,146 @@
+
+The staking and un-staking actions are governed by the following formulas:
+
+**Staking:**
+
+![](http://latex.codecogs.com/gif.latex?s&space;=&space;t&space;*&space;\(\(1&space;&plus;&space;d&space;/&space;b\)^r-1)
+
+**Unstaking:**
+
+![](http://latex.codecogs.com/gif.latex?c&space;=&space;b&space;*&space;\(1&plus;s&space;/&space;t\)^{\frac{1}{r}})
+
+Where `s` is the resulting stake in the court and `d` is the users deposit. When un-staking, `c` is the amount of ANT returned. The other variables represent state and parameters of the staking curve. `r` is the ratio of deposits to total stake, `b` is the balance of ANT in the staking contract, and `t` is the total amount of distributed stake.  
+
+Once a prospective juror has staked they are considered *active* and eligible to be drafted to review cases. A distinction is made between a juror being *staked* and *active.* An *active* juror can *de-activate* themselves at any time, but they will not be able to *un-stake* until any pending disputes they have been involved in have completed.
 
-# 1. Introduction
-The Bitcoin network has proven to be a disruptive force on the global financial system. Bitcoin's innovation is broad but the most significant contributions can be summarized as (1) combining cryptography and economics to create a novel toolkit for solving large scale distributed coordination challenges and (2) using that toolkit to secure transactions on a global peer-to-peer cash system independent of any centralized authority.
+### Juror Drafting
 
-Building on the innovations of Bitcoin, Ethereum created a platform for decentralized applications that abstracts away the crypto-economic security of block production from application developers. By separating concerns, application developers can share a common security layer and thereby pool the cost of securing the network. Pooling the cost of security reduces the portion of value that individual applications must allocate to security compared to running an independent app-chain.
+Juror drafting is managed via a process of *stake-weighted sortition*. In order to manage the sortition process efficiently, all of the court's operations are scheduled into *Terms*. *Terms* are defined in seconds and cannot be changed after the Court has been initialized. Terms are transitioned by calling a public *heartbeat* function which is used to make updates to *active juror stakes* from the preceding term and generate a new *random seed* for use during the subsequent term. A portion of the courts fees are used to compensate the caller of the heartbeat for gas usage.
```

Comment:
The terms cannot be changed at all? Not even by a Network vote? And is this a global setting that applies to all courts, or does it only apply to specific instances of the court?

Reply:
cc: @izqui @bingen not sure about all the considerations related to adjusting this after initialization. 

As far as whether it applies to all courts, it would only apply per instance.

Reply:
After some recent changes the duration of terms could be modified without it being a major issue, but I would keep it static for simplicity. Also the term duration is just the minimum unit of account for time in the Court and the duration in terms for the different states of a dispute (number of terms to commit, number of terms to reveal, etc) can be modified with a network vote.

Reply:
Moving the conversation to #26